buildSrc/src/main/java/org/opensearch/gradle/zipplugin/ZipPublishExtension.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.gradle.zipplugin;
+
+public class ZipPublishExtension {
+
+    private String zipGroup = "org.opensearch.plugin";
+
+    public void setZipgroup(String zipGroup) {
+        this.zipGroup = zipGroup;
+    }
+
+    public String getZipgroup() {
```

Comment:
getZipGroup

Reply:
Will push this change in my next commit.